project/config/settings/test.py

```diff
@@ -27,11 +25,6 @@
     }
 }
 
-# PASSWORDS
-# ------------------------------------------------------------------------------
-# https://docs.djangoproject.com/en/dev/ref/settings/#password-hashers
-PASSWORD_HASHERS = ["django.contrib.auth.hashers.MD5PasswordHasher"]
```

Comment:
The reason I've done this is because the passwords in the fixture files were encrypted using the default hasher, so this was causing all the auth-based tests to fail (the hashes didn't match). This didn't manifest when running the tests with django's own runner because it wasn't set up to use this file at all - it was using `local.py`.

Another approach would have been to rebuild all the fixtures, but this was easier (and makes a minimal difference to test suite speed). If you end up switching all your tests to create test data on-the-fly using `FactoryBoy` or whatever, you could re-enable this micro-optimisation at a later date.

Reply:
We should have a community discussion about which hashing method we want to use for passwords in the DB - we're using **Argon2** ATM (recommended in the Django docs - but not the default, which is PBKDF2), which is why the fixutres have the hashes they do.  So I recommend using `django.contrib.auth.hashers.Argon2PasswordHasher` so that the two match.